### ISSUE TITLE
AutoFit for Column Width

### ISSUE THREAD

**Issue by maui87** [mini-software/MiniExcel#379]
### Excel Type

- [X] XLSX
- [ ] XLSM
- [ ] CSV
- [ ] OTHER

### Description

In other Projects like closedXML there is a function like AutoFit the ColumnWidth to the Content.
In ClosedXML it is called AdjustToContents.

Do you have plans to implement it in MiniExcel? I would be glad to see this feature.

**Comment by A9G-Data-Droid** [comment]
[According to the documentation](https://docs.microsoft.com/en-us/office/troubleshoot/excel/determine-column-widths) the width unit is a single character in the default font. So setting the `column.Width = text.Length` is a good place to start. If you want to support arbitrary fonts, it's going to be much more work.

So you need to loop through all the values in each column finding the `Math.Max` and then pass that one as the width directly.

**Comment by anteeek** [comment]
@A9G-Data-Droid that's a very good point, but it's not satisfactory when using templated excel
However in that case you can set the width of the column manually in the template to match the expected width, which might work for most use cases

**Comment by A9G-Data-Droid** [comment]
@anteeek The people having this issue, like me, have widths that are dynamic. There is no "expected width" because it's different every time. It's an option we are asking for. You don't need to use the AutoFit option if you don't need it.

**Comment by anteeek** [comment]
@A9G-Data-Droid yes I know, but in some use clases the variability of the width is low
Now, I'm not a maintainer, but I think this feature might be surprisingly hard to implement since in similar libraries (including in other languages, for example Exceljs) I have yet to see it implemented, despite large demand for it
